rules/module_extensions.bzl

```diff
@@ -0,0 +1,43 @@
+"""Definitions for bzlmod module extensions."""
+
+load(
+    "//rules:repositories.bzl",
+    "rules_ios_dependencies",
+    "rules_ios_dev_dependencies",
+)
+load(
+    "//tools/toolchains/xcode_configure:xcode_configure.bzl",
+    _xcode_configure = "xcode_configure",
+)
+
+def _non_module_deps_impl(_):
+    rules_ios_dependencies(
+        load_bzlmod_dependencies = False,
```

Comment:
FYI - This confused me, because I thought "load_bzlmod_dependencies" means "dependencies that bzlmod needs" rather than "dependencies that bzlmod has already supplied". Just putting this here in case anyone else stumbles across this, and as always, naming is hard.

Reply:
I had to update it the other day and I felt the same way lol

Happy to change this to something more clear